db/liquibase/changesets/20180915_collection.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="avram" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+    <changeSet logicalFilePath="avram" author="ansingh" id="collection11">
+        <createTable tableName="COLLECTION">
+            <column name="id" type="BIGINT" autoIncrement="true">
+                <constraints primaryKey="true"/>
+            </column>
+            <column name="name" type="VARCHAR(254)">
+                <constraints nullable="false" unique="true"/>
+            </column>
+            <column name="samResource" type="VARCHAR(254)">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
```

Comment:
It would be good to add audit columns to this table. Something like:
```
creator VARCHAR(255) not null
createdTimestamp timestamp not null
deletedTimestamp timestamp
```
And come to think of it collections are mutable right? So maybe we'd want `updatedBy` and `updatedTimestamp` as well.

Reply:
Agreed for createdBy and createdTimestamp, but I'm not sure if we're persisting deleted collections in the db or if we're just deleting them from the db? I thought we were starting out just clean deleting them and not persisting and then adding that when if people expressed a desire for it.

Reply:
Updated can be non-null and be initially populated with the same values as created. Avoids having to conditionally look at the created fields if you want to know the last write.